.github/workflows/main.yml

```diff
@@ -17,18 +17,22 @@ env:
 
 jobs:
   ubuntu-gcc:
-    name: ubuntu-${{ matrix.cc }}
+    name: ubuntu-${{ matrix.cc }}-C++${{ matrix.cppVersion }}
     runs-on: ubuntu-18.04
     strategy:
       matrix:
         cc: [gcc-10, gcc-7]
+        cppVersion: [17, 20]
         include:
           - cc: gcc-7
             cxx: g++-7
             packages: gcc-7 g++-7
           - cc: gcc-10
             cxx: g++-10
             packages: gcc-10 g++-10
+        exclude:
+          - cc: gcc-7
+            cppVersion: 17   
```

Comment:
20?

Reply:
Yes that was the original idea. Somewhere along the way I must have done the exact opposite.